.github/workflows/pre-merge-ci.yaml

```diff
@@ -0,0 +1,25 @@
+---
+name: Pre-merge CI
+
+on:
```

Comment:
I think we want to run this again on the on `push` to verify that after a PR is merged, no issues are introduced.

Reply:
Good point. I did a sneaky rewrite and force push.